geomesa-quickstart-kafka/src/main/resources/log4j.properties

```diff
@@ -5,12 +5,15 @@ log4j.rootLogger=info, stdout
 log4j.logger.geomesa=info
 log4j.logger.org.locationtech.geomesa=info
```

Comment:
Same here?

Reply:
i think info is ok for everything except C*. we should probably modify the C* code to log at debug level tbh...